sbc_compassion/models/import_letters_history.py

```diff
@@ -470,8 +480,14 @@ def _find_language(self, img, pattern_center, template):
         if pattern_center is None:
             return
         # get position of the blue corner
-        bluecorner = bcf.BlueCornerFinder(img)
+        box = [float(
+            self.env['ir.config_parameter'].get_param('bluecorner_x_min')),
+               float(
```

Comment:
./sbc_compassion/models/import_letters_history.py:485:16: E126 continuation line over-indented for hanging indent